RFCs/Image Filter.md

```diff
@@ -0,0 +1,156 @@
+### Context:
+
+When kpack produces a build, certain attributes are surfaced to the end-user that serve to provide additional context about the build and why it was initiated.   Some of this metadata intends to provide end-users with a bill of materials about the build.  Such fields include `Image`, `Pod Name`, `Builder`, `Run Image`, `Source`, `Url`, and `Revision`.  
```

Comment:
Just to avoid confusion with the actual build Bill of Materials, Let's not call this information a bill of materials.